master/getting-started/bare-metal/bare-metal.md

```diff
@@ -762,9 +763,9 @@ EOF
 ```
 
 After defining host endpoints for each node, you should find that internal
-cluster communications are all still working as normal - for example, that you
+cluster communications are all still working as normal—for example, that you
```

Comment:
I did not know this was a thing-at all. I thought I knew hyphens, but it turns out I don't know hyphens.

Reply:
It's all explained in the excellent book the _Chicago Manual of Style_ .... hyphens, en dashes, and em dashes....very interesting topic!!!! 😹 Something else to note: in the case of our particular renderer, we just paste in the actual em dash character, whereas in other markdown situations, you can sometimes use `---` or `&mdash`For us, it's just the character itself. :))))